### ISSUE TITLE
storybook/story-exports rule doesn't support named exports list syntax

### ISSUE THREAD

**Issue by stephenhmarsh** [storybookjs/eslint-plugin-storybook#57]
Hello, apologies if this is not a bug but it sure seems like one to me.

**Describe the bug**
`eslint-plugin-storybook` is throwing `error The file should have at least one story export  storybook/story-exports` when using "named export list" syntax.

Example:

This fails:

```js
const quotesList = () => {
...
};

export { quotesList };
```

This passes:

```js
export const quotesList = () => {
...
};
```

**To Reproduce**
* You could clone: https://github.com/metmuseum/marble/commit/38a1e98095f0142c4ad5a067bb56e154ba419aa5 - this commit introduces the eslint storybook plugin
* Install deps
* Run `npm run lint`

**Expected behavior**
The [named exports _list_ ](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export) syntax should be supported and this rule should not throw any errors on the example above?

**Screenshots**

**Additional context**
This seems to have come from https://github.com/storybookjs/eslint-plugin-storybook/issues/42 and https://github.com/storybookjs/eslint-plugin-storybook/pull/44 so cc @yannbf  - thanks.

**Comment by yannbf** [comment]
Hey @stephenhmarsh thanks for opening this issue! Although the export list syntax is not our recommended way to define stories, this is definitely a bug in the rule as it's supposed to check whether you are exporting stories or not. Thanks for the PR as well! It's people like you who make OSS great <3
